tests/actor/test_power_distributing.py

```diff
@@ -396,7 +397,7 @@ async def test_power_distributor_invalid_battery_id(
         result: Result = done.pop().result()
         assert isinstance(result, Error)
         assert result.request == request
-        err_msg = re.search(r"^No battery 100, available batteries:", result.msg)
+        err_msg = re.search(r"No battery 100, available batteries:", result.msg)
```

Comment:
Why is the `^` removed? It doesn't look like the message changed.

```suggestion
        err_msg = re.search(r"^No battery 100, available batteries:", result.msg)
```

Reply:
If you look at the corresponding commit,  https://github.com/frequenz-floss/frequenz-sdk-python/pull/562/commits/de1331d6bbc6adbca8dafd1198534456e087c91d, `check_request` call is moved downwards,  and so this error is caught and raised by a different `send_result` which wraps an error,  so the string is wrapped by quotes.

Reply:
Ah, ok, It was hidden in the diff, expanding it now I can see it, thanks for the clarification!